docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -148,3 +172,13 @@ userStatus="$(docker exec -u root -it wmagent sh -c "passwd -S $wmaUser" | awk '
 if [ "${userStatus:0:1}" == "P" ]; then
     docker exec -u root -it wmagent sh -c "echo $wmaUser:$wmaUser | chpasswd"
 fi
+
+# Install s-nail
```

Comment:
This indeed  would be better to go into the Docker file, either `wmagent` or `wmbase`. Which might also alleviate/remove  the need for the package configuration here.